smart_pet_buddy/lib/register_page.dart

```diff
@@ -8,6 +8,8 @@ import 'package:flutter/material.dart';
 import 'package:firebase_auth/firebase_auth.dart';
 import 'package:flutter_signin_button/button_builder.dart';
 
+import 'bottomnavbar.dart';
```

Comment:
It looks good 👍 , just wondering why do  we import bottomnavbar here?

Reply:
If authentication is successful, it navigates to Bottom bar, that holds all tabs together, if not brings you back to the landing page

Reply:
ok, thank you!